cmd/installer/subcommands/bootstrap/command.go

```diff
@@ -32,6 +41,25 @@ type cliParams struct {
 	version string
 }
 
+var (
+	installScriptParamsFile = "/etc/datadog-agent/datadog-installer.ini"
```

Comment:
Do we have any plan to support Windows/OSX at some point ? Should we prepare for this and move this to a `command_linux.go` file or something similar ?

Reply:
Eventually it will but it's hard to know in advance if this particular file will be used there as the install_script isn't a concept that exists on windows. I made it slightly more generic with `path.DefaultConfPath` in any case